src/Stratis.SmartContracts.Core/SmartContractTransactionContext.cs

```diff
@@ -56,13 +43,11 @@ public uint Nvout
         }
 
         /// <inheritdoc />
-        public IEnumerable<byte> ContractData
+        public byte[] Data
```

Comment:
Isn't `Data` like a reserved keyword or something? 😋 I think we can come with a better name? I personally thought ContractData wasnt too bad?

Reply:
Maybe SerializedData ?